web/docs/usage/run-pipelines/README.md

```diff
@@ -20,44 +20,55 @@ The diagram below illustrates these mechanics:
 
 ![Pipeline in the Browser](pipeline-browser.excalidraw.svg)
 
-For example, write [`version`](../../operators/version.md) and click *Run* to
-see a single event arrive.
+For example, write [`version`](../../tql2/operators/version.md) and click *Run*
+to see a single event arrive.
 
 ## On the command line
 
 On the command line, run `tenzir <pipeline>` where `<pipeline>` is the
 definition of the pipeline.
 
-If the pipeline expects events as its input, an implicit `load - | read json`
-will be prepended. If it expects bytes instead, only `load -` is prepended.
-Likewise, if the pipeline outputs events, an implicit `write json | save -` will
-be appended. If it outputs bytes instead, only `save -` is appended.
+If the pipeline expects events as its input, an implicit `load_stdin |
+read_json` will be prepended. If it expects bytes instead, only `load_stdin` is
+prepended. Likewise, if the pipeline outputs events, an implicit `write_json |
+save_stdout` will be appended. If it outputs bytes instead, only `save_stdout`
+is appended.
```

Comment:
This is fine because we do want to add it in the short-term, but note that `load_stdin`, `save_stdout`, and `save_stderr` are not yet implemented.

Reply:
Yep, I'm very well aware of it. But since that's on the menu, I thought it's better than removing the text entirely and remembering to bring it back in.